spec/core/ics-002-client-semantics/README.md

```diff
@@ -337,6 +337,16 @@ Once misbehaviour is detected, clients SHOULD be frozen so that no future update
 A permissioned entity such as a chain governance system or trusted multi-signature MAY be allowed
 to intervene to unfreeze a frozen client & provide a new correct ClientMessage which updates the client to a valid state.
 
+#### Retrieve Client Status 
+
+Status is an opaque function defined by a client type to retrieve the current clientState. Status can be either Active, Expired, Unknown or Frozen. 
+
+The function is defined as:
+
+```typescript
+type Status = (Client) => Void
```

Comment:
SHouldn't the return value not be Void? Since it is returning a status

Reply:
Uh, definitively should not be void. @AdityaSripal would be correct to return `bytes` or shall we define a `type Status = bytes` in the Data Structure section and return `Status` itself?